src/source/Ice/IceAgentStateMachine.c

```diff
@@ -383,10 +383,14 @@ STATUS executeGatheringIceAgentState(UINT64 customData, UINT64 time)
             }
             CHK_STATUS(turnConnectionStart(pIceAgent->pTurnConnection));
 
+            turnServerCount++;
+
             break;
         }
     }
 
+    CHK(pIceAgent->iceTransportPolicy != ICE_TRANSPORT_POLICY_RELAY || turnServerCount > 0, STATUS_ICE_MISSING_TURN_SERVER_IN_TURN_ONLY_MODE);
```

Comment:
I don't think we need this check `new RTCPeerConnection({iceTransportPolicy: 'relay'})` doesn't error out in the browser.

I need to confirm, but the following *might* be true.
* The browser might still be gathering+sending connectivity checks (but just not emitting host/prflx candidates)
* There might be a case where relayonly (but no TURN server is given is useful) I can't think of any, but just better not to deviate IMO.